portafly/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10-alpine as builder
+WORKDIR /tmp/portafly
+COPY . ./
+ENV PUBLIC_URL="/portafly"
+RUN yarn install \
+    && yarn build --production --frozen-lockfile
+
+FROM node:10-alpine
```

Comment:
Oh it is a multi stage build.
Be aware It is not supported by Red Hat productization build